homeworkW11D02/README.md

```diff
@@ -0,0 +1,23 @@
+#NETWORKING
```

Comment:
Having in mind all the things you learned so far, code below should be structured better.

You should never mix GUI code with networking code etc.

Client/Server implementations should be a separate class each and than just used in the GUI components. All the complexity could be hidden from the GUI classes.